packages/expo-module-template/package.json

```diff
@@ -1,20 +1,35 @@
 {
   "name": "expo-module-template",
   "version": "2.0.0-rc.1",
-  "description": "Expo Module Template standalone module",
-  "main": "index.js",
+  "description": "ExpoModuleTemplate standalone module",
+  "main": "build/index.js",
+  "types": "build/index.d.ts",
+  "scripts": {
+    "build": "expo-module build",
+    "clean": "expo-module clean",
+    "test": "expo-module test",
+    "postinstall": "expo-module postinstall",
+    "prepare": "expo-module prepare",
+    "prepublishOnly": "expo-module prepublishOnly",
+    "expo-module": "expo-module"
```

Comment:
I'm a bit worried about this - `expo-module-scripts` will work only if you put this module into `expo` repo as it can be installed only through yarn workspaces. This package is not going to be published to npm 😐